src/tests/Loader/classloader/regressions/GitHub_74792/GitHub_74792.cs

```diff
@@ -0,0 +1,35 @@
+using System;
```

Comment:
This is the type of test that can have a nice description and details on what it is testing. Simply naming it with an issue name isn't helpful when trying to understand what it is testing or why it might have failed.

Reply:
Aren't there existing static abstract tests? That is where this test should go.

Reply:
I have added a code comment describing the purpose of the test (the aspect that used to fail). Frankly speaking I'm however still convinced that the GitHub issue database is the most precise and detailed source of information about the issue, its dating and investigation.

Reply:
> I'm however still convinced that the GitHub issue database is the most precise and detailed source of information about the issue, its dating and investigation.

This has been disproven by history. .NET has had multiple bug databases and even today there are issues in the multiple regression folders that have these bug/issue numbers. The above defense was given for all of those decisions too and at this point many/most of them are unrecoverable for these details. The JIT has a slightly different angle, and we can discuss that, but things like this that involve well-defined type system behavior that are clearly bugs in the implementation should have clear, well-defined test coverage.

Reply:
Sounds reasonable to me, thanks for sharing the background. Please let me know if you find the description sufficient or whether you'd prefer more detail.